lib/sepa-xml.js

```diff
@@ -105,7 +105,8 @@ SepaXML.prototype = {
       bic: payment.bic,
       senderName: payment.name,
       recipientIBAN: payment.iban,
-      paymentDescription: payment.description
+      paymentDescription: payment.description,
+      InstructionIdentification: payment.instruction
```

Comment:
Thanks for the PR! Kind of a nitpick, but could you change the first letter to match the camel casing in the rest of the object for consistency's sake?